src/transformState.js

```diff
@@ -5,7 +5,37 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const removeKeyFromStateObj = (key) => {
+    delete state[key];
+  };
+
+  for (const action of actions) {
+    if (Object.hasOwnProperty.call(action, 'type')) {
```

Comment:
Remove this as you have default case

Reply:
how can I be sure that this key('type') exist in nested object?